backend/app/__init__.py

```diff
@@ -12,6 +25,164 @@ def home():
 @app.route("/health")
 def health():
     return "Works"
+  
+@app.route("/login")
+def login():
+    auth_url = createSpotifyOAuth().get_authorize_url()
+    return redirect(auth_url)
+
+@app.route("/authorize")
+def authorize():
+    return 'authorize'
+
+def createSpotifyOAuth():
+    return SpotifyOAuth(
+        client_id=''.join(CLIENT_ID),
+        client_secret=''.join(CLIENT_SECRET),
+        redirect_uri=url_for('authorize', _external=True),
+        scope='playlist-modify-public user-library-read user-library-modify user-read-email user-read-private',
+    )
+    
+@app.route("/imageToEmotion", methods=['POST'])
+def imageToEmotion():
+    """
+    Detect emotion in image
+    ___ 
+    post:
+        form-data:
+        - image : jpg file
+
+        response:
+            200:
+                content: [emotion, score]
+            500 if model fails to detect / interval server errore 
+
+
+    """
+    try:
+        image = request.files.get('image')
+        result = getEmotion(image)
+        if result:
+            return jsonify(result), 200
+        else:
+            return "No result available", 500
+    except Exception as e:
+            return str(e), 500
+
+
+#The following code is done with the help of this link:
+# https://www.youtube.com/watch?v=xdq6Gz33khQ
+class SpotifyAPI(object):
+    access_token = None
+    access_token_expires = datetime.datetime.now()
+    access_token_did_expire = True
+    client_id = None
+    client_secret = None
+    token_url = "https://accounts.spotify.com/api/token"
+    
+    def __init__(self, client_id, client_secret, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.client_id = client_id
+        self.client_secret = client_secret
+
+    def get_client_credentials(self):
+        """
+        Returns a base64 encoded string
+        """
```

Comment:
Inconsistent docstrings. Do it for all the functions.

Reply:
Done 🚀